CHANGELOG.md

```diff
@@ -32,6 +39,7 @@ Changes:
 New features:
 
 * New `TruffleRuby::ConcurrentMap` data structure for use in [`concurrent-ruby`](https://github.com/ruby-concurrency/concurrent-ruby) (#2339, @wildmaples).
+* Add a new `--warn-truffle-regex-fallback` option to see when TRegex cannot compile a regex (#2376, @LillianZ).
```

Comment:
This option has been replaced by `--warn-truffle-regex-compile-fallback` and `--warn-truffle-regex-match-fallback` now.
Not sure how to handle that nicely in the changelog.

Also it's kind of expert experimental options so I wouldn't list them under `New features:`.
We could add it under `Performance:` instead I think.
Or potentially skip it from the changelog since for now it's meant mostly as an internal option.

Reply:
Did the rename come after the 21.2.0 release? If so, then the entry is still accurate for the release. But, I can just remove it too.

Reply:
I would just remove the line, it's an experimental expert option, so not ready to be in the changelog as a user-facing thing yet.